frontend/src/components/activity/ActivityResponsibles.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <e-select
+    v-model="selectedCampCollaborations"
+    :items="availableCampCollaborations"
+    :loading="isSaving || isLoading ? 'secondary' : false"
+    :name="$tc('entity.activity.fields.responsible')"
+    :error-messages="errorMessages"
+    outlined
+    :filled="false"
+    dense
+    multiple
+    chips
+    deletable-chips
+    small-chips
+    v-bind="$attrs"
+    @input="onInput" />
+</template>
+
+<script>
+import serverErrorToString from '@/helpers/serverErrorToString.js'
+
+export default {
+  props: {
+    activity: {
+      type: Object,
+      required: true
+    }
+  },
+  data () {
+    return {
+      oldSelectedCampCollaborations: [],
+      selectedCampCollaborations: [],
+      errorMessages: [],
+      isSaving: false
+    }
+  },
+  computed: {
+    isLoading () {
+      return this.campCollaborations._meta.loading || this.activityResponsibles._meta.loading
+    },
+    availableCampCollaborations () {
+      return this.campCollaborations.items.filter(cc => {
+        return (cc.status === 'established') || (this.currentCampCollaborationIRIs.includes(cc._meta.self))
```

Comment:
Why this filtering? Why can't I newly select an inactive collaboration?

Reply:
more important for 'invited' collaboration, +1

Reply:
Didn't change anything on this one:
https://github.com/ecamp/ecamp3/pull/2369/files/5f65ef05fb914975a5aa1d446a3e62bbe6730e20#diff-6871d9e09bcb7821fc2bace1f9cc77981812808ad4e5d4d8e83565b5e7a87811L198

The idea is/was that only established camp collaborations can be selected and - at the same time - ensure, that people who left the camp are still visible, but cannot be selected anymore.

Should we extend this to invited? 
Only risk I could potentially see is that you invite the wrong person, already select him/her for many activities and then have to change them later. But probably the smaller problem that the fact that you cannot allocate any activities to someone who doesn't manage to accept the invitation.

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/2369/commits/cd87f228268eafa80b06d56dee5ba3d52b8bfe38
If user is not yet registered, email address will be displayed instead of displayName.

Reply:
Könnte das umfallen, wenn ein email invited user ablehnt?